XML/TestCases/FunctionalTests.xml

```diff
@@ -56,12 +84,29 @@
         <TestParameters>
             <param>vCPU=4</param>
         </TestParameters>
-        <Platform>HyperV,Azure</Platform>
+        <Platform>Azure,HyperV</Platform>
         <Category>Functional</Category>
         <Area>CORE</Area>
         <Tags>cpu</Tags>
         <Priority>2</Priority>
     </test>
+    <test>
+        <testName>RELOAD-MODULES-SMP</testName>
+        <setupScript>.\TestScripts\Windows\SETUP-Change-CPU.ps1</setupScript>
+        <testScript>RELOAD-MODULES.ps1</testScript>
+        <files>.\TestScripts\Linux\RELOAD-MODULES.sh,.\TestScripts\Linux\utils.sh</files>
+        <setupType>OneVM</setupType>
+        <OverrideVMSize>Standard_DS4_v2</OverrideVMSize>
+        <TestParameters>
+            <param>VCPU=8</param>
+            <param>LoopCount=100</param>
+        </TestParameters>
+        <Platform>Azure,HyperV</Platform>
```

Comment:
This one looks like a stress test, is it more proper to put it under stress category?

Reply:
this is more on the regression/functional side.
We actually have another TC in Stress xml with reboot loop to 1000, and that's with single core.